### PR TITLE
WIP: 107 Speech recognition NNCF quantization

### PR DESCRIPTION
**Pull request by aleksandr-mokrov**

### CODE REVIEW

**Comment by eaidova** [comment]
@aleksandr-mokrov please resolve conflicts

File: notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb [hunk 1]
```diff
@@ -6,13 +6,13 @@
    "metadata": {},
```
Comment (eaidova) [inline notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb:6]: <p><s>The optimization process contains the following steps:</s></p><p>sentence not bind with the next text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (eaidova) [inline notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb:6]: <pre class="ql-syntax" spellcheck="false">Line #1.    import nncf
</pre><p>please provide some comments regarding API usage in code or in markdown. Especially interesting to explain about advanced parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (eaidova) [inline notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb:6]: <p>using <code>serialize</code>,not save_model  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (eaidova) [inline notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb:6]: <p>why benchamrking time different for fp16 and int8?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

File: notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-wav2vec2.ipynb [hunk 2]
```diff
@@ -8,12 +8,12 @@
    },
```
Comment (eaidova) [inline notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-wav2vec2.ipynb:8]: <p>Compare Accuracy of the original PyTorch, original OpenVINO and Quantized Models -&gt; Compare Accuracy of original PyTorch model, OpenVINO FP16 and INT8 models</p><p> </p><p>This sounds more clear</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (eaidova) [inline notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-wav2vec2.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #91.                                         distance[i][j - 1] + 1,
</pre><p>logically, this part should be moved after quantization, in section where you check models accuracy, also some comments about what is it should be return back</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (eaidova) [inline notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-wav2vec2.ipynb:8]: <p>the same comment about more info about API usage and parameters like in data2vec notebook need to be added</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (eaidova) [inline notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-wav2vec2.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #1.    predicted_text = MetricWER.decode_logits(predictions)
</pre><p>it is slightly strange that we use metric for postprocessing. Can we split were calculation and postprocessing step?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

File: notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb (hunk 1 above)
Comment (eaidova) [inline notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb:6]: <pre class="ql-syntax" spellcheck="false">Line #1.    !pip install -q soundfile
</pre><p>please make sure that librosa does not need anymore. I added it sometime ago due to problem with dataset reading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (eaidova) [inline notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb:6]: <p>please update notebook title and probably readme file also should be updated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Review by eaidova — APPROVED** [review]
